<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>catalogo_wndow</class>
 <widget class="QMainWindow" name="catalogo_wndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catalogo prodotti</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="title_horizontal_layout">
      <item>
       <widget class="QLabel" name="lbl_title">
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Catalogo dei prodotti</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="title_btn_horizontal_layout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_open_carrello">
          <property name="text">
           <string>Vai al carrello</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_finalizza_ordine">
          <property name="text">
           <string>Finalizza ordine</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="tutorial_vertical_layout">
      <item>
       <widget class="QLabel" name="lbl_tutorial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Clicca su un prodotto per poterne conoscere i dettagli o per aggiungerlo al carrello degli acquisti.</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="table_vertical_layout">
      <item>
       <widget class="QTableWidget" name="table_catalogo">
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Titolo prodotto</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quantit√† disponibile</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prezzo cadauno</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="catalogo_btn_horizontal_layout">
      <item>
       <widget class="QPushButton" name="btn_visualizza_dettagli">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Visualizza dettagli</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_aggiungi_al_carrello">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Aggiungi al carrello</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search_prod">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ricerca prodotto</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
